packages/cli/src/commands/build.ts

```diff
@@ -1,16 +1,54 @@
 import { Command } from '@oclif/core'
+import chalk from 'chalk'
 import { spawnSync } from 'child_process'
-import { tmpDir } from '../utils/directory'
+import { existsSync } from 'fs'
+import { copySync, removeSync } from 'fs-extra'
+import { tmpDir, userDir } from '../utils/directory'
 import { generate } from '../utils/generate'
 
 export default class Build extends Command {
   async run() {
     await generate({ setup: true })
+    spawnSync(`yarn build`, {
+      shell: true,
+      cwd: tmpDir,
+      stdio: 'inherit',
+    })
 
-    return spawnSync(`yarn build`, {
-     shell: true,
-     cwd: tmpDir,
-     stdio: 'inherit',
-   })
+    await copyResource(`${tmpDir}/.next`, `${userDir}/.next`)
+    await copyResource(
+      `${tmpDir}/faststore.config.js`,
+      `${userDir}/faststore.config.js`
+    )
+    await copyResource(`${tmpDir}/public`, `${userDir}/public`)
+    await copyResource(`${tmpDir}/vtex.env`, `${userDir}/vtex.env`)
+    await copyResource(
+      `${tmpDir}/lighthouserc.js`,
+      `${userDir}/lighthouserc.js`
+    )
+    await copyResource(`${tmpDir}/cypress`, `${userDir}/cypress`)
+    if (existsSync(`.next/standalone`)) {
```

Comment:
I'm not getting why this is needed. I read the references you linked in the PR description, but isn't standalone mode supposed to have everything that's needed for it to run? Why do we need to copy those files into the folder?

Reply:
yes, the standalone mode is supposed to have everything. But it didn't work as expected. Only after copying the `node_modules`. Back then I read some GH threads and the copy was suggested in this case.